core/common/src/main/java/alluxio/shell/SshCommand.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.shell;
+
+import alluxio.util.ShellUtils;
+
+import javax.annotation.concurrent.NotThreadSafe;
+
+/**
+ * Object representation of a remote shell command by SSH.
+ */
+@NotThreadSafe
+public class SshCommand extends ShellCommand {
+  protected String mHostName;
+
+  /**
+   * Creates a SshCommand instance from the remote hostname and command.
+   *
+   * @param hostname the remote target hostname
+   * @param execString the command to execute
+   */
+  public SshCommand(String hostname, String[] execString) {
+    super(execString);
+    mHostName = hostname;
+    mCommand = new String[]{"bash", "-c",
```

Comment:
Can we just use this for the `super` constructor? (and hopefully keep `mCommand` `final`)

Reply:
`mCommand` and `mHost` are now changed to `private final`